probe-rs/src/flashing/flasher.rs

```diff
@@ -769,7 +770,7 @@ impl<'probe, O: Operation> ActiveFlasher<'probe, O> {
             // Periodically read RTT.
             self.read_rtt()?;
 
-            std::thread::sleep(Duration::from_millis(1));
+            thread::sleep(Duration::from_millis(1));
```

Comment:
To be fair, I like the fully written out for better. I don't really like half imports if they are not required.